### PR TITLE
MM-48510 - fix file limit exceeded alert in dark theme

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
Fix file limit exceeded alert in dark theme.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48510

#### Related Pull Requests
n/a

#### Screenshots
Captura de Pantalla 2022-11-22 a las 19 29 23
Captura de Pantalla 2022-11-22 a las 19 29 53

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by neallred** [comment]
I think this work was already taken up here: https://github.com/mattermost/mattermost-webapp/pull/11631

**Comment by pvev** [comment]
> I think this work was already taken up here: #11627

oh yes, closing this PR then. Thanks @neallred

File: components/file_limit_sticky_banner/index.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ border: 1px solid #FFBC1F;
 border-radius: 4px;
 background: linear-gradient(0deg, rgba(255, 212, 112, 0.16), rgba(255, 212, 112, 0.16)), #FFFFFF;
 padding: 16px 17px;
-color: var(--center-channel-color);
+color: var(--center-channel-text);
```
Comment (AshishDhama) [inline components/file_limit_sticky_banner/index.tsx:45]: @pvev Although we have closed this pr, as you were using `-center-channel-color` earlier, it is the right color, we don’t know `--center-channel-text` got introduced but I think we need to get rid of this.

Comment (pvev) [reply components/file_limit_sticky_banner/index.tsx:45]: ok, but we need to keep in mind that one is used in several places.

**Pull request closed without merging**